src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -197,8 +197,8 @@ def get_attrs(
             )
             result[item]["activations"] = serialize(activations, **kwargs)
 
-            actor = ActorTuple.from_id(user_id=item.user_id, team_id=item.team_id)
-            if actor:
+            if item.user_id or item.team_id:
+                actor = RpcActor.from_id(user_id=item.user_id, team_id=item.team_id)
```

Comment:
Wondering if the interface for `from_id` should just return `None` if both ids are `None`?